src/dbally/views/sqlalchemy_base.py

```diff
@@ -15,28 +15,42 @@ class SqlAlchemyBaseView(MethodsBaseView):
     """
 
     def __init__(self, sqlalchemy_engine: sqlalchemy.Engine) -> None:
+        """
+        Creates a new instance of the SQL view.
+
+        Args:
+            sqlalchemy_engine: SQLAlchemy engine to use for executing the queries.
+        """
         super().__init__()
+        self.select = self.get_select()
         self._sqlalchemy_engine = sqlalchemy_engine
-        self._select = self.get_select()
-        self._where_clause: Optional[sqlalchemy.ColumnElement] = None
 
     @abc.abstractmethod
     def get_select(self) -> sqlalchemy.Select:
-        r"""
-        Creates the initial
-        [SqlAlchemy select object
-        ](https://docs.sqlalchemy.org/en/20/core/selectable.html#sqlalchemy.sql.expression.Select)
-        which will be used to build the query.
         """
+        Creates initial SELECT statement for the view.
```

Comment:
nit: why remove the link? Since we ask people to create sql alchemy statements I believe it's important that we give them link to resources about creating them

Reply:
I agree, but this link was badly formatted because of the line limitation and did not render correctly. I decided to remove it because if the user wants to use the SQL alchemy view, he should know the basic sqlalchemy API.

Reply:
I definitely don't agree with "if the user wants to use the SQL alchemy view" - this is currently the only way to use db-ally with an SQL database that we support, many users coming to db-ally will have to learn both at the same time, and we should provide them the resources to do so.

Reply:
It seems to render correctly in my tests. What should I do to reproduce the problem?

Reply:
I rechecked and now I don't have problem with this, so I think I can reverse the change here